GetIntoTeachingApiTests/Jobs/LocationBatchJobTests.cs

```diff
@@ -23,7 +24,7 @@ public class LocationBatchJobTests : DatabaseTests
         private readonly IMetricService _metrics;
         private readonly Mock<ILogger<LocationBatchJob>> _mockLogger;
 

```

Comment:
Feel like I should have added a new test for the location source such as RunAsync_InsertsLocationsWithSourceAsCsv, but the shared fixture context is making that difficult, so is it okay to just be added to the existing test?

Reply:
Yep existing test is fine, the current test checks the postcode/lat/lng all get set correctly so I think it makes sense to add the check for the source in there as well